.github/workflows/test_notebook.yml

```diff
@@ -4,13 +4,15 @@ on:
   pull_request:
     branches: [main]
     paths:
-      - '**.py'
-      - '**.ipynb'
+     - "pyproject.toml"
+     - "tests/**.py"
+     - "pymc_marketing/**"
```

Comment:
But we also need to be triggered if the notebooks from the docs change themselves, right?

Reply:
Good point. Should I include changes to just tests?

- \"pyproject.toml\" # Environment changes (the ones used in CI)
- \"pymc_marketing/**\" # Changes to module
- \"docs/**.ipynb\" # Changes to notebooks in the documentation

Reply:
Good point. Should I include changes to just tests?

- \"pyproject.toml\" # Environment changes (the ones used in CI)
- \"pymc_marketing/\\*\\*\" # Changes to module
- \"docs/**.ipynb\" # Changes to notebooks in the documentation

Reply:
Good point. Should I include changes to just tests?

- \"pyproject.toml\" # Environment changes (the ones used in CI)
- \"pymc_marketing/\\*\\*\" # Changes to module
- \"docs/\\*\\*.ipynb\" # Changes to notebooks in the documentation

Reply:
maybe add explicitly `"docs/source/notebooks/mmm` (and clv and others). The reason is that we want to omit the dev notebooks https://github.com/pymc-labs/pymc-marketing/tree/main/docs/source/notebooks/clv/dev . Or any other way we make sure we omit just these dev ones